roles/common/files/letsencrypt-gencert

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+service apache2 stop
```

Comment:
No, I don't think so. This gencert script runs only conditionally, if there isn't an existing cert. Regardless, we want to be sure that Apache is running, and we can let Ansible take care of that. If it's already running, the Ansible task won't do anything, so there's no harm done.